torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10552,7 +10552,7 @@ def ref_pairwise_distance(input1, input2):
                         # NumPy's heaviside promotes bool to float16
                         DecorateInfo(unittest.expectedFailure,
                                      'TestBinaryUfuncs',
-                                     'test_reference_numerics_heavisidel',
+                                     'test_reference_numerics',
```

Comment:
This seems like a typo from #69439 cc @mruberry. Without this, the test is not properly skipped.